includes/class-woothemes-sensei-emails.php

```diff
@@ -330,6 +332,34 @@ function teacher_started_course( $learner_id = 0, $course_id = 0 ) {
 		}
 	}
 
+    /**
+     * teacher_completed_lesson()
+     *
+     * Send email to teacher on student completing lesson
+     *
+     * @access public
+     * @return void
+     * @since 1.9.0
+     */
+    function teacher_completed_lesson( $learner_id = 0, $lesson_id = 0 ) {
+        global $woothemes_sensei;
```

Comment:
@mahangu in future I'd like us to use Sensei() to access the global Sensei object